draft-ietf-tokbind-https-06.xml

```diff
@@ -937,6 +965,42 @@ contexts. Other approaches are possible, but are outside the scope of this speci
           mechanisms.
         </t>
       </section>
+        <section title="Correlation" anchor="Correlation">
+          <t>
+            An application's communicating parties, that receive 
+            Token Bindings for TLS connections other than their own, 
+            obtain information about the application's other TLS 
+            connections.
+            The Token Binding IDs in these other Token Bindings can 
+            serve as correlation handles for
+            the endpoints of the other connections.
+            If the application is already aware of these other 
+            connections, then no additional information is being exposed. 
```

Comment:
thx, good question.  I suspect that what we meant to say in the last sentence above is..
> If the endpoints are already aware of these other connections, then no additional information is being exposed.

Does that make more sense?